terraform/services.tf

```diff
@@ -21,6 +21,10 @@ locals {
     CSRF_AUTH_KEY = "secret://${google_secret_manager_secret_version.csrf-token-version.id}"
   }
 
+  session_config = {
+    COOKIE_KEYS = "secret://${google_secret_manager_secret_version.cookie-encryption-key-version.id},secret://${google_secret_manager_secret_version.cookie-hmac-key-version.id}"
```

Comment:
Hash is first, encryption is second: https://github.com/gorilla/securecookie#examples

Reply:
typical off by one error., fixed